docs/docs/platforms/openshift/OPENSHIFT.md

```diff
@@ -0,0 +1,100 @@
+# Deploy To OpenShift
+
+!!! warning "Support Disclaimer"
+    Helm is a Kubernetes package manager that orchestrates the provisioning of applications onto existing Kubernetes infrastructure. The requirements for this infrastructure are described in [Prerequisites](../../userguide/PREREQUISITES.md). The Kubernetes cluster remains your responsibility; we do not provide direct support for Kubernetes or the underlying hardware it runs on.
+
+    If you have followed our documentation on how to configure the Helm charts, and you're using correctly created components, we will then provide support if you encounter an error with installation after running the `helm install` command.
+
```

Comment:
1. "the Helm charts". I think you can remove the "the".
2. I'd change "using correctly created components" (due to the ambiguity of "correctly" being wrongly associated with "using") to either "using correctly-created components" (hyphen added, which clears up the ambiguity) or "components that are correctly created".